features/search-input-type.yml.dist

```diff
@@ -14,4 +14,15 @@ status:
     safari: ≤5
     safari_ios: ≤4.2
 compat_features:
+  # baseline: high
```

Comment:
This notably does not say "Same status as overall feature" because the `use_ranges_before_baseline_low_date` logic only runs for the overall feature. @ddbeck I see a few options:

- Add a `useRangesBeforeBaselineLowDate` parameter to `computeBaseline`. This would fix the problem locally, but users of `computeBaseline` don't know what to pass in. You shouldn't really be able to pick if you want the ranges or not as a consumer, that's something we do as providers of the data.
- Revert https://github.com/web-platform-dx/web-features/pull/1208 to not block progress on this more important PR and worry about it later.

WDYT?

Reply:
I've sent https://github.com/web-platform-dx/web-features/pull/1286.

Reply:
I agree on the revert and I just approved it. I need to think on the broader problem. I think doing https://github.com/web-platform-dx/web-features/issues/1211—which would just make ranges a dumb fact of the status calculation—is the right way to go, but that's more of a guess right now.